package.json

```diff
@@ -116,7 +116,8 @@
     "audit": "yarn npm audit --recursive --environment production --severity moderate --ignore '@metamask/types (deprecation)'",
     "download-builds": "tsx .devcontainer/download-builds.ts prep-build",
     "download-builds:test": "tsx .devcontainer/download-builds.ts prep-build-test",
-    "master-sync": "node development/master-sync.js"
+    "master-sync": "node development/master-sync.js",
+    "update-mock-cdn": "node test/e2e/mock-cdn/update-mock-cdn-files.js"
```

Comment:
Hey @seaona : is this script also called somewhere ?

Reply:
hey Jyoti, the script is not called anywhere in the code for now. We can use run it locally whenever we want to update/get new files, instead of doing this manually. For example, for adding this new spec, where we test a data upgrade
https://github.com/MetaMask/metamask-extension/issues/23641

In the future we could think on having some periodic upgrades of the mock files in the case we want, and we could use this script too.